contracts/src/v0.8/EIP_712_1014_4337/EIP_712_1014_4337.t.sol

```diff
@@ -0,0 +1,270 @@
+
+    bytes signature = // Signature by LINK_WHALE. Signs off on "hi" being set as the greeting on the Greeter.sol contract, via their SCA.
+        hex"cc503bc5125e8bc43fdeb28931df74c718b433f1b0bec332ffa90bb7be71425b2cc23dc7174253ee4fe29107b38cb589c4fe97ada812381b41b284a0e24c862101";
```

Comment:
to confirm my understanding, does this signature get verified inside `validateUserOp()`?

Reply:
Yep this is what gets verified in validateUserOp